tests/gcp/operators/test_bigquery_system.py

```diff
@@ -17,40 +17,32 @@
 # specific language governing permissions and limitations
 # under the License.
 """System tests for Google Cloud Build operators"""
-import unittest
 
 from tests.gcp.operators.test_bigquery_system_helper import GCPBigQueryTestHelper
-from tests.gcp.utils.base_gcp_system_test_case import SKIP_TEST_WARNING, TestDagGcpSystem
 from tests.gcp.utils.gcp_authenticator import GCP_BIGQUERY_KEY
+from tests.test_utils.gcp_system_helpers import GCP_DAG_FOLDER, provide_gcp_context, skip_gcp_system
+from tests.test_utils.system_tests_class import SystemTest
 
 
-@unittest.skipIf(TestDagGcpSystem.skip_check(GCP_BIGQUERY_KEY), SKIP_TEST_WARNING)
-class BigQueryExampleDagsSystemTest(TestDagGcpSystem):
+@skip_gcp_system(GCP_BIGQUERY_KEY, require_local_executor=True)
```

Comment:
This parameters should not have `require_local_executor` set to `True`. I have problem with running this test.

Reply:
Yes, I agree. Can you make a PR with fix because I am on holidays?

Reply:
Sorry for late response. I will try to fix it but I have limited time due to my holidays :)